test/functional/admin/dashboard_controller_test.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+module Admin
+  class DashboardControllerTest < ActionController::TestCase
+
+    setup do
+      @user = users(:user1)
+      sign_in AdminUser.create!(email: 'admin@example.com', password: 'password')
+    end
+
+    test "test index method with call_number descending param" do
+      get :index, params: {order: 'call_number_desc'}
+      assert_response :success
+    end
+
+    test "test index method with p_call_number_desc param" do
+      get :index, params: {order: 'p_call_number_desc'}
+      assert_response :success
+    end
+
+    test "test index method with p_call_number_asc param" do
+      get :index, params: {order: 'p_call_number_asc'}
+      assert_response :success
```

Comment:
So these tests don't make sure that the sort is actually sorting, right?  Their purpose is to make sure the method itself is returning a success value?